lib/qless/server.rb

```diff
@@ -152,6 +152,14 @@ def strftime(t)
           formatted
         end
       end
+
+      def h(text)
+        Rack::Utils.escape_html(text)
+      end
+
+      def hattr(text)
```

Comment:
I'm not able to find the usage of this method

And looks like we can add tests [here](https://github.com/wallarm/qless/blob/16dd11aae62bd975111f9ee374d3c070ef1077e6/spec/integration/server_spec.rb#L390)

Reply:
I added both methods just because it was recommended here: http://sinatrarb.com/faq.html#escape_html . I can remove unused #hattr if you wish :)

Reply:
Add spec and removed unused data